Extensions/DeviceSensors/devicesensortools.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * @memberof gdjs.evtTools
+ * @class devicesensors
+ * @static
+ * @private
+ */
+
+gdjs.evtTools.devicesensors = {};
```

Comment:
nitpicking: you can also declare an object like this:

```
gdjs.evtTools.devicesensors = {
  orientation: {
    _isActive: false,
    _absolute: 0,
    _alpha: 0;
    _beta: 0;
    _gamma: 0
  }
};
```

Slightly easier to type and read - but it's nitpicking, it's also good how you've done.